include/h2o/version.h

```diff
@@ -27,17 +27,17 @@
 #endif
 
 #ifdef H2O_GITREV
-#define H2O_VERSION "2.3.0-DEV@" H2O_TO_STR(H2O_GITREV)
+#define H2O_VERSION "2.3.0-beta1@" H2O_TO_STR(H2O_GITREV)
```

Comment:
This file is automatically generated using the values of CMake, hence we do not need to do anything.

Reply:
Understood. I re-generated this file with the updated version values that you pointed out.